### DISCUSSION TITLE
Can I use this dataset with Tortoise?

### DISCUSSION THREAD

**Discussion by shalevc1098** [neonbjb/tortoise-tts#714] [Q&A]
Hello, I came across [this dataset](https://huggingface.co/ivrit-ai) for the Hebrew language and wondered if I could somehow integrate it with Tortoise. Thanks for helping!

**Comment by thewh1teagle** [comment]
@shalevc1098 
Hey!
I'm working on that with ivrit.ai dataset, if you interested I created a repo and put there the research details / source code etc.
Do you still interested in it?

https://github.com/thewh1teagle/israwave
